tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,134 @@
+"""
+.. _tut_events_and_annotation_objects:
+
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the glossary.
```

Comment:
Would be good to add a RST link to the glossary

Reply:
yes but I did not find how. I don't think is referenceable yet.